components/eamxx/tests/uncoupled/surface_coupling/surface_coupling.cpp

```diff
@@ -16,6 +17,112 @@
 
 namespace scream {
 
+constexpr Real test_tol = std::numeric_limits<Real>::epsilon()*1e4;
+
+// Test function for prescribed values
+Real test_func(const int col, const int t) {
+  return (Real)(col + 1 + t);
```

Comment:
Minor: the (C style) cast is not needed. The compiler already does that for you.